ygen/protogen_test.go

```diff
@@ -267,9 +273,29 @@ func TestGenProtoMsg(t *testing.T) {
 			continue
 		}
 
-		if !protoMsgEq(got, tt.wantMsg) {
-			diff := pretty.Compare(got, tt.wantMsg)
-			t.Errorf("%s: genProto3Msg(%#v, %#v, *genState): did not get expected protobuf message definition, diff(-got,+want):\n%s", tt.name, tt.inMsg, tt.inMsgs, diff)
+		seenMsg := map[string]bool{}
+		for _, w := range tt.wantMsgs {
+			seenMsg[w.Name] = false
```

Comment:
optional: could call this notSeen map[string]interface{} instead, and delete the keys as you go along. Seems a bit more intuitive, but up to you.

Reply:
Done, that is cleaner :+1: